### PR TITLE
Add python library pycloudstack

### PR DESCRIPTION
**Pull request by kenplusplus**
pycloudstack is a python library to abstract common operations in
hypervisor/kvm or kubernetes based cloud stack. It can be used to
create test, deployment and automation tool for specifc cloud stack
like SGX/TDX/AMX etc.

Signed-off-by: Lu Ken <ken.lu@intel.com>

### CODE REVIEW

**Review by clsulliv — APPROVED** [review]
Fixed some grammar issues in the README. Otherwise, LGTM.

File: utils/pycloudstack/pycloudstack/vmm.py [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+            return VM_STATE_SHUTDOWN
+        return None
+
+    def get_ip(self, force_refresh=False):
+        """
+        Get VM available IP on virtual or physical bridge
+
+        In some situation, a wrong IP may be parsed before the formal IP allocated.
+        This issue causes following operations, fail with the bad IP
+        This issue is traced in BLR-560 (https://jira.devtools.intel.com/browse/BLR-560)
```
Comment (jialeif) [inline utils/pycloudstack/pycloudstack/vmm.py:338]: Internal link should be hidden.  Keep BLR-560

File: utils/pycloudstack/pycloudstack/vmguest.py [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+
+        ssh_port = None
+        ssh_ip = None
+
+        # use *(tnow - tstart) < timeout * check to ensure it really elapsed *timeout* seconds
+        while ((tnow - tstart) < timeout and not ssh_ok):
+            try:
+                # In some situation, VMMLibvirt may get a wrong IP before the formal IP allocated
+                # The issue will lead to following ssh check failure if keep using the bad IP
+                # This issue is traced in BLR-560 (https://jira.devtools.intel.com/browse/BLR-560)
```
Comment (jialeif) [inline utils/pycloudstack/pycloudstack/vmguest.py:165]: Internal url should not be exposed

**Review by jialeif — APPROVED** [review]
LGTM. It's better to remove internal Jira link.

File: utils/pycloudstack/pycloudstack/vmguest.py (hunk 2 above)
Comment (kenplusplus) [reply utils/pycloudstack/pycloudstack/vmguest.py]: Thanks! and Fixed

**Comment by kenplusplus** [comment]
> Fixed some grammar issues in the README. Otherwise, LGTM.

Thanks!

**Pull request merged**